examples/nft-nd/nft.yml

```diff
@@ -15,5 +15,5 @@ events:
         type: ByteArray
 permissions:
   - hash: fffdc93764dbaddd97c48f252a53ea4643faa3fd
-    methods: ["update", "destroy"]
+    methods: ["update", "destroy","setRoyaltyInfo"]
```

Comment:
Revert it. This section allows contract to call native Management's `destroy` and `update` methods, it's not related to the contract itself.